eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/eventlist/view/EventListToolbar.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2016 SchedJoules
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.schedjoules.eventdiscovery.eventlist.view;
+
+import android.content.res.Resources;
+import android.support.v4.app.FragmentActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+
+import com.schedjoules.eventdiscovery.R;
+import com.schedjoules.eventdiscovery.common.BaseActivity;
+
+
+/**
+ * The toolbar of the Event list screen.
+ *
+ * @author Gabor Keszthelyi
+ */
+public final class EventListToolbar
+{
+    private final Toolbar mToolbar;
+    private final Listener mListener;
+
+
+    public EventListToolbar(Toolbar toolbar, Listener listener)
+    {
+        mToolbar = toolbar;
+        mListener = listener;
+    }
+
+
+    public void initToolbar(FragmentActivity fragmentActivity)
```

Comment:
With `EventListToolbar` and `EventListView` (this name may not be the best) the motivation was to create an object kind of declaratively for a UI item, so it can contain and hide its corresponding boilerplate and initialization code. It is very specific to the actual UI component on the actual screen, so that's why I didn't create an interface. It would feel strange for me to create an interface for such a specific and simple classes. The only possible general method I can think of might be `init(Activity)`.
I usually only create interface for these kind of specific classes when they grow bigger, to have an overview what they do by looking at the interface.
But of course I can create the interfaces, let me know if I should, or if you have any other idea in mind how to handle this.